lib/jxl/enc_fast_lossless.h

```diff
@@ -46,9 +47,15 @@ struct JxlFastLosslessFrameState;
 // Returned JxlFastLosslessFrameState must be freed by calling
 // JxlFastLosslessFreeFrameState.
 JxlFastLosslessFrameState* JxlFastLosslessPrepareFrame(
-    const unsigned char* rgba, size_t width, size_t row_stride, size_t height,
-    size_t nb_chans, size_t bitdepth, int big_endian, int effort,
-    void* runner_opaque, FJxlParallelRunner runner);
+    JxlChunkedFrameInputSource input, size_t width, size_t height,
+    size_t nb_chans, size_t bitdepth, int big_endian, int effort);
+
+class JxlEncoderOutputProcessorWrapper;
```

Comment:
Can you remove these if FJXL_STANDALONE is 1? also from the cc file

Reply:
In general, could you test that the scripts in tools/fast_lossless still work?

Reply:
They are quite broken currently, starting with fatal error: 'jxl/encode.h' file not found

Reply:
no, I don't think so, I will open an issue for this

Reply:
Opened https://github.com/libjxl/libjxl/issues/2959